multiqc/modules/odgi/odgi.py

```diff
@@ -112,84 +120,70 @@ def parse_odgi_stats_report(self, f):
         # Add the data to the main odgi_stats_map dict
         if s_name in self.odgi_stats_map:
             log.debug("Duplicate sample name found! Overwriting: {}".format(s_name))
-        self.odgi_stats_map[s_name] = self.compress_stats_data(data)
+        self.odgi_stats_map[s_name] = data_flat
 
     def plot_general_odgi_stats(self):
         """
         Plot the odgi stats by adding them to the general statistics table.
         """
         headers = OrderedDict()
-        headers["Length"] = {
+        headers["length"] = {
             "title": f"{config.base_count_prefix} Length",
             "description": f"Graph length in nucleotides ({config.base_count_desc} )",
             "scale": "BuPu",
             "modify": lambda x: x * config.base_count_multiplier,
             "shared_key": "base_count",
         }
-        headers["Nodes"] = {
+        headers["nodes"] = {
             "title": "Nodes",
             "description": "Number of nodes in the graph.",
             "scale": "OrRd",
             "format": "{:,.0f}",
         }
-        headers["Edges"] = {
+        headers["edges"] = {
             "title": "Edges",
             "description": "Number of edges in the graph.",
             "scale": "PuBu",
             "format": "{:,.0f}",
         }
-        headers["Paths"] = {
+        headers["paths"] = {
             "title": "Paths",
             "description": "Number of paths in the graph.",
             "scale": "Greens",
             "format": "{:,.0f}",
         }
-        headers["Components"] = {
+        headers["components"] = {
             "title": "Components",
-            "description": "Number of weakly connected components in the " "graph.",
+            "description": "Number of weakly connected components in the graph.",
             "scale": "Oranges",
             "format": "{:,.0f}",
         }
-        headers["A"] = {
-            "title": "A",
-            "description": "Number of adenine bases in the graph.",
+        headers["pct_gc"] = {
```

Comment:
I understand the need to have less columns. But for debugging and comparing in detail across several different graphs, I would still like to keep a column for each base. Small example:

![image](https://user-images.githubusercontent.com/10184961/124248603-4ac9a780-db23-11eb-915e-2416cb8674e8.png)

Here, we see that the total number of `N`s changed a lot (problem of PGGB), so I want to see this in the report. If I see percentages, it does not really help me, because the number of bases generally changed.

I will be a bad guy and change this back because of the above. If you don't agree, then the percentage wins, obviously.

Reply:
In the end, the exact number of bases helps to debug and validate the outcome of the PGGB pipeline.

Reply:
Couldn't you have a separate table for the very same purpose that holds all your columns of interest? I get what Phil means that otherwise the general stats table gets very large when running more than 2-3 tools that MultiQC supports in a pipeline.

Reply:
Ah, now I get it. I was not aware somehow that this general stats table incorporates statistics from all ran MultiQC tools.

We need an extra table, then. So for the general table I can imagine `length`, `nodes`, `edges`, `paths`.
All other statistics can go into an extra table. However, would it be fine, if all of these 4 basic statistics are there, again?
It would be better to have all of these in one table, to get the full picture of what is going on.

Reply:
What would be your suggestion @apeltzer @ewels ?

Reply:
Suggestion would be to only keep the 2-3 most interesting cols in the `general_stats` table and add all the other interesting bits to your own table https://multiqc.info/docs/#creating-a-table

Reply:
Yeah was confusing for me too in the beginning but that's why we're all here to help ;-) 

If you think that `length, nodes, edges, paths` are the most interesting then have these in general stats and add the rest to your own custom table (and maybe have those four in your custom table too - not sure about that bit  @ewels knows more about it)

Reply:
Will do an additional table. In a meeting now, will push something as soon as I can.

Reply:
Yup, exactly right 👍🏻 Fine to keep the columns if helpful but split into a module-level table.

Don't _have_ to have columns in General Stats either - only if it's helpful to have these stats alongside other analysis. If it makes sense to keep them all in one table then can just be in a module section. If only running with this module it'll basically look the same as now.